config/settings.yml

```diff
@@ -0,0 +1,41 @@
+project_groups:
+  - name: Prevention
+    body: Inform, reduce, and control the spread of the outbreak
+    project_skills:
```

Comment:
Small thing to reduce confusion: Should rename this `project_types` since that is different than `skills`. Groupings are on the categories and not skills.

Reply:
great find! thanks!